ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -459,3 +459,63 @@ if get_option('enable-mediapipe')
     install_dir: filter_subplugin_install_dir
   )
 endif
+
+if get_option('enable-snpe')
```

Comment:
Use the new "feature" meson feature. Rebase to the recent master HEAD, then you will see it (@zhoonit has updated meson.build accordingly)

Reply:
@myungjoo I updated the meson.build to enable this snpe feature w.r.t. the existence of environment variable `$SNPE_ROOT`. Please check.